solutions/microfrontends/packages/acme-tailwind-config/package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.0.1",
   "repository": "https://github.com/vercel/examples.git",
   "license": "MIT",
-  "private": true,
```

Comment:
What would happen if we keep it? Some dev teams by default wants configs private. Not sure what's the best approach here.

Reply:
It can't be `private` if it's included in changesets.